### PR TITLE
Deprecate SparkTestCase and TestCase for pytest fixture

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: python/tests/parquet/test_resolver.py [hunk 1]
```diff
@@ -12,33 +12,26 @@
...
-import tempfile
-from unittest import TestCase
-
 from rikai.parquet.resolver import Resolver
+from rikai.testing import assert_count_equal
+
 
+def teardown_function(_):
+    Resolver.reset()
+    print("RESET!!")
```
Comment (changhiskhan) [inline python/tests/parquet/test_resolver.py:21]: super nit: remove print

**Review by changhiskhan — APPROVED** [review]
lgtm.

nit: not a release blocker, we could think about using https://pypi.org/project/pytest-spark/ to initialize the test sparkSession. not a blocker tho

File: python/tests/parquet/test_resolver.py (hunk 1 above)
Comment (eddyxu) [reply python/tests/parquet/test_resolver.py]: lol my bad

**Comment by eddyxu** [comment]
Maybe i am too conservative , i hesitate to use project with 500/1000 stars :)

**Pull request merged**